lib/opensusebasetest.pm

```diff
@@ -492,6 +492,18 @@ sub firewall {
     return ($old_product_versions || $upgrade_from_susefirewall) ? 'SuSEfirewall2' : 'firewalld';
 }
 
+=head2 remount_tmp_if_ro
+
+    remount_tmp_if_ro()
+
+Mounts /tmp to shared memory if not possible to write to tmp.
+For example, save_y2logs creates temporary files there.
+
+=cut
+sub remount_tmp_if_ro {
+    script_run 'touch /tmp/test_ro || mount -t tmpfs /dev/shm /tmp';
```

Comment:
Are you sure this mount does the thing you intend? I guess it should be either `-t tmpfs` or `-o bind /dev/shm` but not both

Reply:
I guess you are confused by `-t` having 3 meaning: `--types`, `--source` and `--target`.